Makefile

```diff
@@ -13,7 +13,7 @@ include Makefile.global
 all: extension
 
 # build extension
-extension:
+extension: src/include/citus_version.h
```

Comment:
I changed it to $(citus_abs_srcdir)/src/include/citus_version.h, similar to reference to same directory in [line 25](https://github.com/citusdata/citus/blob/b4839eb7c9db676b5106b40244cfbb3eecec4931/Makefile#L25)